pkg/actuators/machineset/controller_suite_test.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package machineset
```

Comment:
I'm not sure we should just call this stuff 'machineset' here and elsewhere.  Primarily this will add noise to the provider's container and it will confuse users.